test/gtest/ucp/test_ucp_atomic.cc

```diff
@@ -28,6 +30,7 @@ class test_ucp_atomic : public test_ucp_memheap {
         add_variant_with_value(variants, features, variant | UCP_ATOMIC_MODE_CPU, "cpu" + name);
         add_variant_with_value(variants, features, variant | UCP_ATOMIC_MODE_DEVICE, "device" + name);
         add_variant_with_value(variants, features, variant | UCP_ATOMIC_MODE_GUESS, "guess" + name);
+        add_variant_with_value(variants, features, variant | UCP_ATOMIC_MODE_CPU | ODP, "cpu/odp" + name);
```

Comment:
is the mode of cpu atomics+ODP able to reproduce the original issue of KSM+ODP?
seems to me that no, since it will not issue atomic operations on KSM+ODP rkey, even when ODP is supported..
maybe we want to set the mode "guess" and adjust the logic of selecting the atomic mode to also consider ODP?

Reply:
done